README.md

```diff
@@ -186,7 +186,10 @@ This repository contains a curated list of awesome open source libraries that wi
 * [Inference](https://github.com/roboflow/inference) - A fast, production-ready inference server for computer vision supporting deployment of many popular model architectures and fine-tuned models. With Inference, you can deploy models such as YOLOv5, YOLOv8, CLIP, SAM, and CogVLM on your own hardware using Docker.
 * [Jina](https://github.com/jina-ai/jina)  ![](https://img.shields.io/github/stars/jina-ai/jina.svg?style=social) - Cloud native search framework that   supports to use deep learning/state of the art AI models for search.
 * [KServe](https://github.com/kserve/kserve) ![](https://img.shields.io/github/stars/kserve/kserve.svg?style=social) - Serverless framework to deploy and monitor machine learning models in Kubernetes - [(Video)](https://www.youtube.com/watch?v=hGIvlFADMhU).
+* [Langfuse](https://github.com/langfuse/langfuse) ![](https://img.shields.io/github/stars/langfuse/langfuse.svg?style=social) - Langfuse is an open source observability & analytics solution for LLM-based applications.
+* [LangKit](https://github.com/whylabs/langkit) ![](https://img.shields.io/github/stars/whylabs/langkit.svg?style=social) - LangKit is an open-source text metrics toolkit for monitoring language models.
 * [LightLLM](https://github.com/ModelTC/lightllm) ![](https://img.shields.io/github/stars/ModelTC/lightllm.svg?style=social) - LightLLM is a Python-based LLM (Large Language Model) inference and serving framework, notable for its lightweight design, easy scalability, and high-speed performance.
+* [LLMonitor](https://github.com/LLMonitor/llmonitor) ![](https://img.shields.io/github/stars/LLMonitor/llmonitor.svg?style=social) - Observability & analytics for AI apps and agents.
```

Comment:
Seems more relevant to monitoring section

Reply:
![image](https://github.com/EthicalML/awesome-production-machine-learning/assets/8592144/377b3671-9de3-4dc9-9c48-1001bf07f42e)
I know, but the current repo fails to separate monitoring from serving and put them in the same module, shall we make a refactoring for the whole repo afterwards?